src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-information/child-information.js

```diff
@@ -0,0 +1,68 @@
+import { genericSchemas } from '../../../generic-schema';
+import { suffixes } from '../../../constants';
+import { childInfo } from './helpers';
+import { isChapterFieldRequired } from '../../../helpers';
+import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
+import _ from 'lodash/fp';
+
+export const schema = {
+  type: 'object',
+  properties: {
+    childrenToAdd: {
+      type: 'array',
+      minItems: 1,
+      items: {
+        type: 'object',
+        properties: {
+          first: genericSchemas.genericTextInput,
+          middle: genericSchemas.genericTextInput,
+          last: genericSchemas.genericTextInput,
+          suffix: {
+            type: 'string',
+            enum: suffixes,
+          },
+          ssn: genericSchemas.genericNumberAndDashInput,
+          birthDate: genericSchemas.date,
+        },
+      },
+    },
+  },
+};
+
+export const uiSchema = {
+  childrenToAdd: {
+    'ui:options': {
+      itemName: 'Child',
+      viewField: childInfo,
+    },
+    items: {
+      first: {
+        'ui:title': 'Child’s first name',
+        'ui:required': formData => isChapterFieldRequired(formData, 'addChild'),
+      },
+      middle: {
+        'ui:title': 'Child’s middle name',
+      },
+      last: {
+        'ui:title': 'Child’s last name',
+        'ui:required': formData => isChapterFieldRequired(formData, 'addChild'),
+      },
+      suffix: {
+        'ui:title': 'Child’s suffix',
+        'ui:options': {
+          widgetClassNames: 'usa-input-medium',
+        },
+      },
+      ssn: {
+        'ui:title': 'Child’s Social Security Number',
+        'ui:options': {
+          widgetClassNames: 'usa-input-medium',
+        },
+        'ui:required': formData => isChapterFieldRequired(formData, 'addChild'),
+      },
+      birthDate: _.merge(currentOrPastDateUI("Child's date of birth"), {
```

Comment:
Related to my comment above, it'd be better to try and import `merge` directly, instead of the entire lodash library. Alternatively, I _think_ the `spread` operator could be used here without harm, since `currentOrPastDateUI` doesn't have a `ui:required` property on it, there shouldn't be a risk of overriding anything.